app/models/attribute/ConfigTable.scala

```diff
@@ -0,0 +1,204 @@
+package models.attribute
+
+import models.daos.slick.DBTableDefinitions.{DBUser, UserTable}
+import models.utils.MyPostgresDriver.simple._
+import play.api.Play.current
+import scala.slick.lifted.ForeignKeyQuery
+
+case class ConfigApi(apiAttributeCenterLat: Double, apiAttributeCenterLng: Double,
+                     apiAttributeZoom: Double, apiAttributeLatOne: Double, apiAttributeLngOne: Double, apiAttributeLatTwo: Double,
+                     apiAttributeLngTwo: Double, apiStreetCenterLat: Double, apiStreetCenterLng: Double, apiStreetZoom: Double,
+                     apiStreetLatOne: Double, apiStreetLngOne: Double, apiStreetLatTwo: Double, apiStreetLngTwo: Double,
+                     apiRegionCenterLat: Double, apiRegionCenterLng: Double, apiRegionZoom: Double, apiRegionLatOne: Double,
+                     apiRegionLngOne: Double, apiRegionLatTwo: Double, apiRegionLngTwo: Double)
+
+case class Config(openStatus: String, mapathonEventLink: Option[String], cityCenterLat: Double, cityCenterLng: Double, southwestBoundaryLat: Double,
+                  southwestBoundaryLng: Double, northeastBoundaryLat: Double, northeastBoundaryLng: Double, defaultMapZoom: Double,
+                  tutorialStreetEdgeID: Int, offsetHours: Int, excludedTags: String, databaseName: String, configApi: ConfigApi)
+
+class ConfigTable(tag: slick.lifted.Tag) extends Table[Config](tag, Some("sidewalk"), "config") {
+  def openStatus: Column[String] = column[String]("open_status", O.NotNull)
+  def mapathonEventLink: Column[Option[String]] = column[Option[String]]("mapathon_event_link")
+  def cityCenterLat: Column[Double] = column[Double]("city_center_lat", O.NotNull)
```

Comment:
Let's just use `Float` for these... Mostly because we are using floats everywhere else and because we aren't entering anything with nearly that precision into this table anyway. There may have been other reasons to use floats that I don't recall, so probably best to stick with it here for now.

Reply:
So to clarify, for all columns that are double in ConfigTable, change them to floats correct?

Reply:
No you can leave them as Doubles in the actual table, it's just in the Scala where we're changing them. This is how we're doing it elsewhere

Reply:
Ultimately glad that you didn't change this because I have now finally been burned by using `Float` instead of `Double`, so I'm glad we're using `Double`.